src/Files.App/Views/Properties/GeneralPage.xaml.cs

```diff
@@ -77,7 +77,7 @@ bool GetNewName(out string newName)
 				return false;
 			}
 
-			bool SaveDrive(DriveItem drive)
+			bool SaveDrive(LocatableDriveItem drive)
```

Comment:
What does `Locatable` represent?

Reply:
The storage abstraction we are going to introduce has ILocatable class and this class implements that class. Locatable means that the class has path, name, and other info, which are all necessary when you are searching the folder on the filesystem.

Reply:
@d2dyno1 are you on board with this?

Reply:
Renamed followings to have Locatable.

- ILocatableSideBarItem
- LocatableWSLItem
- LocatableDriveItem
- LocatableStorageItem